src/NServiceBus.Core/v6-obsoletes.cs

```diff
@@ -968,4 +971,25 @@ public void DoNotRequireExplicitRouting()
             throw new NotImplementedException();
         }
     }
+}
+
+namespace NServiceBus.Settings
+{
+    using System;
+
+    public partial class ScaleOutSettings
+    {
+        [ObsoleteEx(TreatAsErrorFromVersion = "6", RemoveInVersion = "7", ReplacementTypeOrMember = "Default starting with V6")]
```

Comment:
ReplacementTypeOrMember is for the actual TypeOrMember not text. use `Message` instead